src/components/timecard/edit-shift-hours/EditShiftHours.spec.jsx

```diff
@@ -405,11 +424,12 @@ describe('EditShiftHours', () => {
         fireEvent.click(saveButton);
       });
 
-      await waitFor(() => {
-        const errorMessage = screen.queryByText(
-          'You must enter a finish time in the HH:MM 24 hour clock format'
-        );
-        expect(errorMessage).toBeFalsy();
+      await expectNeverToHappen(() => {
+        expect(
+          screen.getByText(
+            'Enter a finish time in the 24 hour clock format, for example, 08:00 or 0800'
+          )
+        ).toBeInTheDocument();
```

Comment:
Why the change from waitFor and expect(errorMessage).toBeFalsy()?

Reply:
This test wasn't actually working, me and Joanna tried a few things to get the test working and found this was the best way to test something not appearing on the screen.

Reply:
Ok makes more sense, some sort of test library hell